llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -2826,6 +2843,8 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
     return selectExtInst(ResVReg, ResType, I, CL::s_clamp, GL::SClamp);
   case Intrinsic::spv_wave_active_countbits:
     return selectWaveActiveCountBits(ResVReg, ResType, I);
+  case Intrinsic::spv_wave_any:
+    return selectWaveActiveAnyTrue(ResVReg, ResType, I);
```

Comment:
Why can’t you just do:

`return selectWaveNOpInst(ResVReg, ResType, I, SPIRV::OpGroupNonUniformAny, 3);`

here?

Reply:
no reason. Its just left over from previous commits. Give me a sec

Reply:
https://github.com/llvm/llvm-project/pull/115902/commits/42d24e47bfd0d635119bfa858bdf01b14238fe8c